.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -8,11 +8,14 @@ assignees: ''
 ---
 # Bug Report
 
+### Is the issue related to model conversion?
+If the ONNX checker reports issues with this model then this is most probably related to the converter used to convert the original framework model to ONNX. Please create this bug in the appropriate converter's GitHub repo (pytorch, tensorflow-onnx, sklearn-onnx, keras-onnx, onnxmltools) to get the best help.
+
```

Comment:
Should you add the actual link for the other repo?

Reply:
I think it is OK to not add the links. These are all git repos + if the issue is related to model conversion it is likely that the OP already knows about this converter.